exercise/exercise_models.py

```diff
@@ -177,6 +177,10 @@ class LearningObject(UrlMixin, ModelWithInheritance):
         verbose_name=_('LABEL_CONTENT'),
         blank=True,
     )
+    consecutive_order = models.IntegerField(
```

Comment:
This feels a little redundant extra DB field in addition to the old 'order' field, although counted differently. I'm wondering if anything would break badly if the behavior of 'order' would be changed to represent "consecutive order". I didn't notice 'order' being used in aplus logic much at all, but of course we don't know if something uses it through API.

Reply:
Ok, with closer reading, we don't want to change the order behavior. Stupid comment from me :-)

Reply:
Yeah, the "order" seems to provide "weights", which determine the ordering; the name of the old "order" is quite misleading. (When I heard about it, I was excited that it might provide a solution, but it didn't take long to realize the value is 0 for nearly everything. :( )

Reply:
One more question: if consecutive_order is only needed for the API, specifically for Jutut. Could it be calculated dynamically when the API response is composed, and not save it to database at all?

Reply:
@PasiSa Why can't we make the order field actually represent the order? I don't see anything blocking that

Reply:
That protocol/aplus.py usage seems to assume that the order actually is the absolute order: it is used to set the html tag id for the exercise. There can now be the same html id on multiple exercises if the order is the same (which is possible e.g. by manually setting the order)